docs/man.md

```diff
@@ -0,0 +1,183 @@
+# qpc-tools
+
+## Name
+
+qpc-tools - Configuration tools for the Quipucords server and command line interface client.
+
+
+## Synopsis
+
+`qpc-tools command subcommand [options]`
+
+## Description
+
+The `qpc-tools` package is used to configure the Quipucords server and command line interface (CLI) client.
+
+# Usage
+
+The following sections describe these commands, their subcommands, and their options in more detail. They also describe additional tasks that are not highlighted in the previous list of major workflow tasks.
+
+## Server
+This section describes various `qpc-tools` commands for installing and configuring the Quipucrods server.
+
+### Server Install Command
+The `qpc-tools server install` command with no options performs a basic installation with the preset defaults. However, it is recommended to run the `qpc-tools server install` command with options to change default username and passwords. The most common scenarios where you might use options to change the basic installation process are explained in the `Installing the server offline`_ and  `Installing a specific version of the server`_ sections.
```

Comment:
The `Installing the server offline` and `Installing a specific version of the server` links are not working